syntax = "proto3";

package audio;

service AudioService {
  rpc StreamAudio (stream AudioData) returns (stream AudioResponse);
}

message AudioData {
  string format = 1;
  int32 sample_rate = 2;
  bytes audio_bytes = 3;
}

message AudioResponse {
  bytes audio_bytes = 1;
}
